dns/dyndns/src/www/services_dyndns_edit.php

```diff
@@ -139,6 +139,10 @@ function is_dyndns_username($uname)
         $input_errors[] = gettext("The username contains invalid characters.");
     }
 
+    if ((string)((int)$pconfig['ttl']) != $pconfig['ttl']) {
+        $input_errors[] = gettext("The TTL value needs to be a valid integer number.");
```

Comment:
Broken.  Not all services use this field.  Those that don't throw a hissy fit and won't allow to save.  e.g. the custom service type.

Reply:
@NOYB @AdSchellevis 
Please check my pull request #1903 (below)